compiler/src/iree/compiler/Dialect/Stream/Transforms/ScheduleAllocation.cpp

```diff
@@ -665,10 +665,21 @@ applyAsyncCollectiveOp(IREE::Stream::AsyncCollectiveOp asyncOp,
   return success();
 }
 
-static LogicalResult applyAsyncBarrierOp(IREE::Stream::AsyncBarrierOp barrierOp,
+static LogicalResult applyAsyncBarrierOp(IREE::Stream::AsyncBarrierOp asyncOp,
```

Comment:
nit: barrierOp was better (there's lots of async ops)

Reply:
Changed back.